lints/lint_dnsname_contains_empty_label.go

```diff
@@ -0,0 +1,53 @@
+package lints
+
+import (
```

Comment:
This doesn't look `go fmt`ed.

Reply:
it is or it wouldn't have passed the `gofmt` test!

Reply:
ok!